tests/agent_features/test_record_llm_feedback_event.py

```diff
@@ -46,12 +45,10 @@ def test_record_llm_feedback_event_all_args_supplied():
     def _test():
         record_llm_feedback_event(
             rating=1,
-            message_id="message_id",
+            trace_id="123456789abcdefgh",
             category="informative",
-            request_id="request_id",
-            conversation_id="conversation_id",
             message="message",
-            metadata={"foo": "bar", "message": "custom-message"},
+            metadata={"foo": "bar", "my-message": "custom-message"},
```

Comment:
Leave this as message - it's intentionally testing that the user can't override the builtins by passing a metadata key of the same name. Perhaps we should add a comment here explaining this so it's more clear?
```suggestion
            metadata={"foo": "bar", "message": "custom-message"},
```